tailcall-http-cache/src/http_cache_manager.rs

```diff
@@ -0,0 +1,152 @@
+use http_cache_reqwest::{CacheManager, HttpResponse, MokaCache};
+use http_cache_semantics::CachePolicy;
+use serde::{Deserialize, Serialize};
+pub type BoxError = Box<dyn std::error::Error + Send + Sync>;
+pub type Result<T> = std::result::Result<T, BoxError>;
+
+use std::sync::Arc;
+
+pub struct HttpCacheManager {
+    pub cache: Arc<MokaCache<String, Store>>,
+}
+
+impl Default for HttpCacheManager {
+    fn default() -> Self {
+        Self::new(MokaCache::new(42))
```

Comment:
Can you create an issue to configure the size of the HTTP cache via an upstream setting.

Reply:
before
```graphql
@upstream(httpCache: true)
```

after
```graphql
@upstream(httpCache: 128)
```

Reply:
sure, will create an issue.